batch7crud-roll9/src/main/java/com/lftechnology/batch7crud/controller/IndexController.java

```diff
@@ -10,11 +10,9 @@
 
 import com.lftechnology.batch7crud.util.TypeCaster;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.lftechnology.batch7crud.util.TypeCaster'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)